powerquery-docs/query-folding-basics.md

```diff
@@ -36,7 +36,7 @@ The previous image emphasizes the applied steps section, which contains the foll
 - **Sorted rows**: Sorts the table using one or more columns.
 - **Kept top rows**: Filters the table to only keep a certain number of rows from the top of the table.
 
-This set of step names is a friendly way to view the M script that Power Query has created for you. There are several ways to view the full M script. In Power Query, you can select **Advanced Editor** in the **View** tab. You can also select **Advanced Editor** from the **Query** group in the **Home** tab. In some versions of Power Query, you can also change the view of the formula bar to show the query script by going into the **View** tab and from the **Layout** group, select **Script view** > **Query script**.
+This set of step names is a friendly way to view the M script that Power Query has created for you. There are several ways to view the full M script. In Power Query, you can select **Advanced Editor** in the **View** tab. You can also select **Advanced Editor** from the **Query** group in the **Home** tab. In some versions of Power Query, you can also change the view of the formula bar to show the query script by going into the **View** tab, and from the **Layout** group, select **Script view** > **Query script**.
```

Comment:
The only way is the Advanced Editor which I am using a lot. There are not many ways, it's basically one step that differs between versions. Full m script = code = advanced editor. Could be simpler here.

Reply:
```suggestion
This set of step names is a friendly way to view the M script that Power Query has created for you. To view the full M script. In Power Query, select **Advanced Editor** in the **View** tab or select **Advanced Editor** from the **Query** group in the **Home** tab. In some versions of Power Query, you can also go to the **View** tab and select **Script view** > **Query script** from the **Layout** group to change the view of the formula bar to show the query script.
```

In some versions of Power Query is still unclear. Which?